vagrant/README.md

```diff
@@ -0,0 +1,138 @@
+Vagrant provider for jclouds
+============================
+
+Building
+--------
+
+1. Vagrant bindings
+  * `git clone https://github.com/neykov/vagrant-java-bindings`
+  * `cd vagrant-java-bindings`
+  * `mvn clean install`
+  * Copy `target/vagrant-java-bindings-0.0.1-SNAPSHOT.jar` to jcloud's classpath
+2. Vagrant provider
+  * `git clone https://github.com/neykov/jclouds-vagrant`
+  * `cd jclouds-vagrant/vagrant`
+  * `mvn clean install`
+  * Copy `target/vagrant-2.0.0-SNAPSHOT.jar` to jcloud's classpath
+
+Using in Apache Brooklyn
```

Comment:
The catch is that Vagrant doesn't have `credential` and `identity`, while Brooklyn requires them. Setting any values will do, but it needs to be there.
Agree it's too user specific, so here's probably not the best place to put it.
Let's keep it around for some more time until moved to a better place.